src/script/components/Modals/PrimaryModal/InputForm/InputForm.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Wire
+ * Copyright (C) 2024 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import type {FormEvent} from 'react';
+
+interface InputFormProps {
+  onSubmit: (event: FormEvent<HTMLFormElement>) => void;
+  inputValue: string;
+  inputPlaceholder: string;
+  onInputChange: (value: string) => void;
+}
+
+export const InputForm = ({onSubmit, inputValue, inputPlaceholder, onInputChange}: InputFormProps) => {
+  return (
+    <form onSubmit={onSubmit}>
+      <label htmlFor="modal-input" className="visually-hidden">
+        {inputPlaceholder}
+      </label>
+
+      <input
+        ref={ref => ref?.focus()}
```

Comment:
I think so, didn't see it before. I'll check that, good catch 💪

Reply:
After adding `autoFocus`, we get an error:


"The autoFocus prop should not be used, as it can reduce usability and accessibility for users.eslint [jsx-a11y/no-autofocus](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/tree/HEAD/docs/rules/no-autofocus.md)"

In this case, it's not a big deal, let's leave it like that